src/node/install.ts

```diff
@@ -103,13 +103,9 @@ export async function downloadBrowser(): Promise<void> {
     if (NPM_NO_PROXY) process.env.NO_PROXY = NPM_NO_PROXY;
 
     function onSuccess(localRevisions: string[]): void {
-      // Use Intel x86 builds on Apple M1 until native macOS arm64
-      // Chromium builds are available.
-      if (os.platform() !== 'darwin' && os.arch() !== 'arm64') {
-        logPolitely(
-          `${supportedProducts[product]} (${revisionInfo.revision}) downloaded to ${revisionInfo.folderPath}`
-        );
-      }
+      logPolitely(
```

Comment:
I think you just need to rebase and remove the line with the os import above. It appears that all tests would pass then.

Reply:
fixed it and solved conflicts